rs/pocket_ic_server/src/lib.rs

```diff
@@ -34,8 +34,9 @@
 //! The start state is a dedicated state that always exists independent of which computations have
 //! been carried out. A state which has no outcoming computations is called a leaf.
 
-mod external_canister_types;
 mod nonmainnet_features;
+
+pub mod external_canister_types;
```

Comment:
The module must be public now that the tests are _integration_ tests (which cannot access private stuff from the crate).